check-code/Makefile

```diff
@@ -0,0 +1,5 @@
+test: .cabal-sandbox
```

Comment:
Although I also checked in the `tinc.freeze` file. If we stick with that we should just add that freeze file to the dependencies and not the hackage index.